bots/bot.go

```diff
@@ -0,0 +1,36 @@
+package bot
+
+import "crypto/tls"
+
+type BotHandleFunc func(string, []byte)
+
+type Bot interface {
+	HandleRegistration(string, []byte) string
+	Run()
+	Stop()
+}
+
+func SendRequestToCONIKS(address string, msg []byte) ([]byte, error) {
+	conf := &tls.Config{
+		InsecureSkipVerify: true,
+	}
+
+	conn, err := tls.Dial("tcp", address, conf)
+	if err != nil {
+		return nil, err
+	}
+	defer conn.Close()
+
+	_, err = conn.Write(msg)
```

Comment:
I just take a look at `dename` and `coname` implementation. It looks like they didn't check this, too. For example:
https://github.com/yahoo/coname/blob/master/httpfront/httpfront.go#L143-L145
https://github.com/andres-erbsen/chatterbox/blob/master/transport/transport.go#L141-L143